javascript-timers/interval.js

```diff
@@ -0,0 +1,11 @@
+var $countdown = document.querySelector('.countdown-display');
+
+function timeout() {
+  $countdown.textContent -= 1;
+  if ($countdown.textContent === '0') {
+    $countdown.textContent = '~Earth Beeeeloooowwwww Us~';
+  }
+}
+
+var i = setInterval(timeout, 1000);
+setTimeout(function () { clearInterval(i); }, 4000);
```

Comment:
you don't want to use setTimeout here, you want to call clearInterval in the function when the countdown is complete

Reply:
![timeout2](https://user-images.githubusercontent.com/113648549/200910064-c2cce229-8e5e-41f4-863e-6e580351d6a9.gif)